lib/modules/platform/gitlab/index.ts

```diff
@@ -690,28 +696,15 @@ async function tryPrAutomerge(
         await setTimeout(mergeDelay * attempt ** 2); // exponential backoff
       }
 
-      // Even if Gitlab returns a "merge-able" merge request status, enabling auto-merge sometimes
-      // returns a 405 Method Not Allowed. It seems to be a timing issue within Gitlab.
-      for (let attempt = 1; attempt <= retryTimes; attempt += 1) {
-        try {
-          await gitlabApi.putJson(
-            `projects/${config.repository}/merge_requests/${pr}/merge`,
-            {
-              body: {
-                should_remove_source_branch: true,
-                merge_when_pipeline_succeeds: true,
-              },
-            },
-          );
-          break;
-        } catch (err) {
-          logger.debug(
-            { err },
-            `Automerge on PR creation failed. Retrying ${attempt}`,
-          );
-        }
-        await setTimeout(mergeDelay * attempt ** 2); // exponential backoff
-      }
+      await gitlabApi.postJson(
+        `projects/${config.repository}/merge_requests/${pr}/notes`,
```

Comment:
which is the minimum gitlab version to support this API?

does this API never return 405? like the merge API?

Reply:
Even if `/notes` has been there for years, how about the inner `/merge` logic - also years?

Reply:
![image](https://github.com/user-attachments/assets/0a9469bd-2abd-4870-8616-f41c8d26df74)
Yes it seems very old